src/sentry/apidocs/examples/project_examples.py

```diff
@@ -187,120 +187,11 @@
         "require2FA": False,
         "avatar": {"avatarType": "upload", "avatarUuid": "24f6f762f7a7473888b259c566da5adb"},
         "features": [
-            "performance-uncompressed-assets-ingest",
-            "dashboards-rh-widget",
-            "performance-db-main-thread-visible",
-            "transaction-name-mark-scrubbed-as-sanitized",
-            "sentry-pride-logo-footer",
-            "issue-list-prefetch-issue-on-hover",
-            "mep-rollout-flag",
-            "performance-issues-m-n-plus-one-db-detector",
-            "session-replay-ui",
-            "alert-crash-free-metrics",
-            "performance-n-plus-one-db-queries-visible",
-            "session-replay-recording-scrubbing",
-            "profile-file-io-main-thread-visible",
-            "performance-consecutive-http-detector",
-            "profiling",
-            "symbol-sources",
-            "advanced-search",
-            "performance-n-plus-one-db-queries-post-process-group",
-            "minute-resolution-sessions",
-            "performance-transaction-name-only-search-indexed",
-            "performance-large-http-payload-detector",
-            "performance-consecutive-db-queries-ingest",
-            "business-to-team-promotion",
-            "shared-issues",
-            "performance-large-http-payload-post-process-group",
-            "promotion-be-adoption-enabled",
+            "global-views",
+            "discover-basic",
+            "incidents",
+            "uptime",
```

Comment:
A lot of these flags are out of date and don't add anything to the example. replaced them with a few `permant.py` flags instead.